apps/web/src/components/PoolCard/PoolCard.tsx

```diff
@@ -62,84 +62,106 @@ const PoolCard = ({ pool, headingLevel = "h3" }: PoolCardProps) => {
 
   return (
     <div
-      data-h2-background-color="base(white) base:dark(black.light)"
-      data-h2-shadow="base(large)"
-      data-h2-margin="base(0, 0, x1, 0)"
-      data-h2-padding="base(x1) p-tablet(x1, x1, x1, x6) l-tablet(x2, x2, x2, x8)"
+      data-h2-background-color="base(foreground)"
+      data-h2-shadow="base(larger)"
+      data-h2-margin-top="base(x1)"
+      data-h2-padding="base(x1) p-tablet(x2, x2, x2, x6.5)"
       data-h2-position="base(relative)"
       data-h2-radius="base(rounded)"
     >
       <div
-        aria-hidden="true"
-        data-h2-position="base(absolute)"
-        data-h2-location="base(0, auto, auto, x.5) p-tablet(0, auto, auto, x2)"
+        data-h2-background-color="base(secondary)"
+        data-h2-position="base(absolute) base:selectors[::before](absolute) base:selectors[::after](absolute)"
+        data-h2-location="base(-x.25, auto, auto, x1) p-tablet(-x.25, auto, auto, x1.5) base:selectors[::before](auto, auto, 0px, 0px) base:selectors[::after](auto, 0px, 0px, auto)"
+        data-h2-radius="base(rounded, rounded, 0px, 0px)"
+        data-h2-height="base(x4.5) base:selectors[::before](0px) base:selectors[:::after](0px)"
+        data-h2-width="base(x3.5) base:selectors[::before](0px) base:selectors[::after](0px)"
+        data-h2-content="base:selectors[::before](' ') base:selectors[::after](' ')"
+        data-h2-display="base:selectors[::before](block) base:selectors[::after](block)"
+        data-h2-border-top="base:selectors[::before](x3 solid transparent) base:selectors[::after](x3 solid transparent)"
+        data-h2-border-bottom="base:selectors[::before](x1.5 solid transparent) base:selectors[::after](x1.5 solid transparent)"
+        data-h2-border-left="base:selectors[::before](x3 solid secondary)"
+        data-h2-border-right="base:selectors[::after](x3 solid secondary)"
+        data-h2-transform="base:selectors[::before](translate(0, 1.5rem)) base:selectors[::after](translate(0, 1.5rem))"
       >
-        <div
-          className="recruitment-flag"
-          data-h2-background-color="base(secondary)"
-          data-h2-padding="base(x2, x.5, x1, x.5)"
-          // data-h2-padding="base(x2, x.5, x2.5, x.5)"
-          // style={{
-          //   clipPath: `polygon(0% 0%, 0% 100%, 50% calc(100% - 2rem), 100% 100%, 100% 0)`,
-          // }}
+        <span
+          data-h2-color="base(black) base:iap(white)"
+          data-h2-font-weight="base(700)"
+          data-h2-font-size="base(h5, 1)"
+          data-h2-layer="base(2)"
+          data-h2-position="base(absolute)"
+          data-h2-location="base(auto, auto, x1.25, 50%)"
+          data-h2-transform="base(translate(-50%, 0px))"
```

Comment:
Something here is causing the flag text to wrap for me 😢 

![Screenshot 2023-10-06 103641](https://github.com/GCTC-NTGC/gc-digital-talent/assets/4127998/8042208b-2838-4cb7-b8f7-19e4dc7a10e2)

Reply:
@esizer Should be fixed now :D